L80-csharp-load-balancing.md

```diff
@@ -1,10 +1,10 @@
-L80: .NET: Load balancing, connectivity and wait for ready in client
+L96: .NET: Load balancing, connectivity and wait for ready in client
```

Comment:
The gRFC number is assigned when the PR is opened, not when it's merged, so this is still L80 -- the number of this gRFC does not change just because other gRFCs started after this one were merged first.  Please put this number back to what it was.

Reply:
Done. I thought the number was updated at the end - https://github.com/grpc/proposal/#process

I'll leave this PR to Jan or yourself to make any final changes before merging.